servers/quarkus-server/src/main/resources/application.properties

```diff
@@ -23,19 +23,24 @@ nessie.server.should-sendstack-trace-to-api-client=true
 nessie.version.store.type=INMEMORY
 
 ## JGit version store specific configuration
-### Which type of jgit version store to use: INMEMORY or DISK. in memory is best for tests and DISK is recommended. Dynamo is an experimental backend
+### Which type of jgit version store to use: INMEMORY or DISK. in memory is best for tests and DISK is recommended. Dynamo and Mongo are experimental backends
```

Comment:
Please update whole comment. INMEMORY and DISK are for test purposes. Dynamo is currently for production uses. Mongo is not yet available.

Reply:
This has been removed as previously discussed.